test/Microsoft.Data.Analysis.Tests/DataFrame.IOTests.cs

```diff
@@ -1171,6 +1171,48 @@ public static IEnumerable<object[]> CsvWithTextQualifiers_TestData()
                     }
                 )
             };
+            yield return new object[] // Quotations in Data
+            {
+                @"Name,Age,Description
+Paul,34,""Paul lives in """"Vermont VA"""".""
```

Comment:
What do you think about using the new `"""` syntax?

https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-11#raw-string-literals

See the first example in that link.

Reply:
Ooh I really like this, makes it much easier to parse the test cases without having to translate the escaped quotes.